src/parser/create_statement.py

```diff
@@ -23,10 +23,12 @@
 
 class ColumnDefinition:
     def __init__(self, col_name: str,
-                 col_type: ParserColumnDataType, col_dim: List[int]):
+                 col_type: ParserColumnDataType, col_dim: List[int],
+                 col_unique_column_constarint=False):
```

Comment:
Rather than adding a new parameter just for unique, we should add a new generic constraint enum parameter. This way we will not have to add a new parameter for each different type of constraint.